### DISCUSSION TITLE
Cannot read properties of undefined (reading '1')

### DISCUSSION THREAD

**Discussion by Aladeenb** [aptos-labs/aptos-developer-discussions#93] [Questions] [answered]
Hi,

I get this error whenever I try to interact with a specific function via the explorer.

The transaction however does not fail when using Aptos CLI (`aptos move run`).

- link to the function:
https://explorer.aptoslabs.com/account/0x91c62bc900d7ab1d4c699b293d82c754a68369af278f7422ef5eadcbfed8efbd/modules/run/core/claim?network=testnet

- the CLI script that works: 
`aptos move run --function-id 0x91c62bc900d7ab1d4c699b293d82c754a68369af278f7422ef5eadcbfed8efbd::core::claim --type-args 0x91c62bc900d7ab1d4c699b293d82c754a68369af278f7422ef5eadcbfed8efbd::core::RenegadeCoin --args "address:0xfdf8c24445a8448cea35388b4018fff140be99d4ef57888d2da8f662afa97d0f" u64:3
`

<img width="389" alt="image" src="https://github.com/aptos-labs/aptos-developer-discussions/assets/48839940/31b07e60-b285-41d2-b05a-e5493ec8f02b">

What can be the cause?

**Comment by 0xmaayan** [comment]
Hi! thanks for reporting! There seems to be a bug in Explorer, the team is working on a fix.

**Reply by Aladeenb** [reply]
Thanks for the quick reply @0xmaayan .

Something that worth mentioning here: one of the function's arguments has the type `Object<LiquidCoinMetadata<LiquidCoin>>` which is basically an object with a nested generic type.

That one was more likely causing the issue.

One solution was to get the object address as input instead of `Object<LiquidCoinMetadata<LiquidCoin>>`, and then use `object::address_to_object` to get the actual object.

**Reply by gregnazario** [reply, marked as answer]
So, actually turns out it was an issue in Petra, which should be updated in the extension now. The nested types should work properly now.

The issue was `Object<LiquidCoinMetadata<LiquidCoin>>` and it did not parse properly the inner type (since really it just matters it's object to know it's an addrss)
